src/main/java/Duke.java

```diff
@@ -1,10 +1,200 @@
-public class Duke {
-    public static void main(String[] args) {
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.time.LocalDate;
+import  java.time.format.DateTimeParseException;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+
+class Duke {
+
+
+    public static ArrayList<Task> tasks;
+
+    public static Deadline createDeadline(String [] input) {
+       String description = "";
+       String by;
+        StringBuilder sb = new StringBuilder();
+        for(int i = 1; i < input.length; i++){
```

Comment:
White spacing is inconsistent here - just some minor fixes with Java Coding Standard should do 👍